### PR TITLE
Code from 1st 4 classes

### PR DESCRIPTION
**Pull request by EllieSky**
Please look at my homework solution

### CODE REVIEW

File: tests/homework_1.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        self.assertEqual(14400, actual_seconds)
+
+    def test_positive_minutes(self):
+        actual_seconds = self.convert_into_seconds(minutes=34)
+        # actual_seconds = convert_into_seconds(0, 34, 0)
+        self.assertEqual(2040, actual_seconds)
+
+    def test_positive_minutes_and_seconds(self):
+        actual_seconds = self.convert_into_seconds(seconds=25, minutes=100)
+        ##### SAME AS:
```
Comment (EllieSky) [inline tests/homework_1.py:19]: Please remove this comment

File: tests/homework_1.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        self.assertEqual(None, actual_seconds)
+
+    def test_negative_input_minutes(self):
+        actual_seconds = self.convert_into_seconds(1, -1, 0)
+        self.assertEqual(None, actual_seconds)
+
+    def convert_into_seconds(hours=0, minutes=0, seconds=0):
+        if seconds < 0 or minutes < 0 or hours < 0:
+            print("Negative input for time units is not allowed, please use a positive number")
+        else:
```
Comment (EllieSky) [inline tests/homework_1.py:38]: This makes no sense, could you name the variables more descriptively

**Review by frolovde — COMMENTED** [review]
Nice! =)

**Review by KatyaGer — APPROVED** [review]

**Comment by richardxudai** [comment]
approval

**Comment by richardxudai** [comment]
approval

**Pull request merged**